apps/cms.c

```diff
@@ -266,6 +266,34 @@ static CMS_ContentInfo *load_content_info(int informat, BIO *in, BIO **indata,
     return NULL;
 }
 
+static void warn_binary(const char *file, int flags)
+{
+    BIO *bio;
+    unsigned char linebuf[1024], *cur, *end;
+    int len;
+
+    if ((flags & CMS_BINARY) != 0)
```

Comment:
Do you mean == 0 here?

Reply:
This logic look like if its not in binary form load the file in binary form??
May need a comment if this is right..

Reply:
Ahh I understand after reading further - it needs a comment.

Reply:
I did mean that no warning is shown if `-binary` is given - see also the warning texts :)

Reply:
Comment(s) added